recipe/meta.yaml

```diff
@@ -1,45 +1,51 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "array_record" %}
+{% set version = "0.7.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
-source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
-
 build:
+  # Skip Windows and macOS x86_64: Google only publishes wheels for Linux and macOS ARM64
+  # Building from source requires Bazel which is not available in conda build environment
```

Comment:
We do have bazel tho: https://anaconda.org/anaconda/bazel

Reply:
If it's a specific version we're missing, I think it's good to explicitly state it in the comment